### PR TITLE
EPIAP to Master

### PR DESCRIPTION
**Pull request by skinkie**

### CODE REVIEW

**Comment by ue71603** [comment]
@skinkie  should we also introduce the OJP like branches at this point?

**Comment by skinkie** [comment]
Yes, for backward fixing etc.

**Review by ue71603 — APPROVED** [review]

**Comment by ue71603** [comment]
@trurlurl @Joostb61 ok for you? Then we merge and release...

File: xsd/NeTEx_publication_timetable.xsd [hunk 1]
```diff
@@ -1713,6 +1731,17 @@ Provides a general purose wrapper for NeTEx data content.</xsd:documentation>
...
 			<xsd:field xpath="@version"/>
 		</xsd:key>
+		<!-- =====AccessEquipment============================ -->
+		<xsd:keyref name="AccessEquipment_AnyKeyRef" refer="netex:AccessEquipment_AnyVersionedKey">
+			<xsd:selector xpath=".//netex:AccessEquipmentRef"/>
+			<xsd:field xpath="@ref"/>
+			<xsd:field xpath="@version"/>
+		</xsd:keyref>
+		<xsd:key name="AccessEquipment_AnyVersionedKey">
+			<xsd:selector xpath=".//netex:EntranceEquipment | .//netex:LiftEquipment | .//netex:LiftCallEquipment | .//netex:RampEquipment |.//netex:StaircaseEquipment   | .//netex:EscalatorEquipment  |  .//netex:TravelatorEquipment  | .//netex:RoughSurface | .//netex:PlaceLighting"/>
```
Comment (skinkie) [inline xsd/NeTEx_publication_timetable.xsd:1741]:
```suggestion
			<xsd:selector xpath=".//netex:EntranceEquipment | .//netex:LiftEquipment | .//netex:LiftCallEquipment | .//netex:RampEquipment | .//netex:StaircaseEquipment | .//netex:EscalatorEquipment | .//netex:TravelatorEquipment  | .//netex:RoughSurface | .//netex:PlaceLighting"/>
```

File: xsd/netex_framework/netex_genericFramework/netex_accessibility/netex_acsb_limitations.xsd [hunk 2]
```diff
@@ -132,14 +150,31 @@ Rail transport, Roads and Road transport
 			<xsd:documentation>Whether a PLACE has lift free access.</xsd:documentation>
 		</xsd:annotation>
 	</xsd:element>
-	<xsd:element name="AudibleSignalsAvailable" type="LimitationStatusEnumeration" default="false">
+	<xsd:element name="RampFreeAccess" type="LimitationStatusEnumeration" default="unknown">
 		<xsd:annotation>
-			<xsd:documentation>Whether a PLACE is wheelchair accessible.</xsd:documentation>
+			<xsd:documentation>Whether a PLACE has ramp free access. Ramp free isn't necessary absolute. If a ramp is not steep, then a PLACE 
+			can still be considered ramp free.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_genericFramework/netex_accessibility/netex_acsb_limitations.xsd:156]:
```suggestion
			<xsd:documentation>Whether a PLACE has ramp free access. Ramp free isn't necessary absolute. If a ramp is not steep, then a PLACE can still be considered ramp free.</xsd:documentation>
```

File: xsd/NeTEx_publication_timetable.xsd (hunk 1 above)
Comment (ue71603) [reply xsd/NeTEx_publication_timetable.xsd:1741]: Interestingly it validated... Sorry I didn't remove the ending of the merge commit right.

File: xsd/netex_framework/netex_genericFramework/netex_accessibility/netex_acsb_limitations.xsd [hunk 3]
```diff
@@ -132,14 +150,31 @@ Rail transport, Roads and Road transport
...
+		</xsd:annotation>
+	</xsd:element>
+	<xsd:element name="LevelAccessIntoVehicle" type="LimitationStatusEnumeration" default="unknown">
+		<xsd:annotation>
+			<xsd:documentation>Whether the platform is high enough and gap is small enough for level access into vehicle. 
+At least at a designated wheelchair door position the gap between platform and vehicle floor (of level access vehicle) does not exceed 75 mm measured horizontally and 50 mm measured vertically including sliding step (according to PRM TSI).</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_genericFramework/netex_accessibility/netex_acsb_limitations.xsd:161]:
```suggestion
			<xsd:documentation>Whether the platform is high enough and gap is small enough for level access into vehicle. At least at a designated wheelchair door position the gap between platform and vehicle floor (of level access vehicle) does not exceed 75 mm measured horizontally and 50 mm measured vertically including sliding step (according to PRM TSI).</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd [hunk 4]
```diff
@@ -242,12 +242,12 @@ Rail transport, Roads and Road transport
...
 			<xsd:enumeration value="intercityRail">
 				<xsd:annotation>
-					<xsd:documentation>Deprecated. Use Mode rail with RailSubmode longDistance.</xsd:documentation>
+					<xsd:documentation>Deprecated. Use mode rail with RailSubmode longDistance.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:enumeration>
 			<xsd:enumeration value="urbanRail">
 				<xsd:annotation>
-					<xsd:documentation>Deprecated. Use Mode metro with MetroSubmode urbanRail (or in some few cases rail with RailSubmode local). </xsd:documentation>
+					<xsd:documentation>Deprecated. Use mode metro with MetroSubmode urbanRail (or in some few cases rail with RailSubmode local). </xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd:250]:
```suggestion
					<xsd:documentation>DEPRECATED. Use mode metro with MetroSubmode urbanRail (or in some few cases rail with RailSubmode local).</xsd:documentation>
```

File: xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd [hunk 5]
```diff
@@ -242,12 +242,12 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="rail"/>
 			<xsd:enumeration value="intercityRail">
 				<xsd:annotation>
-					<xsd:documentation>Deprecated. Use Mode rail with RailSubmode longDistance.</xsd:documentation>
+					<xsd:documentation>Deprecated. Use mode rail with RailSubmode longDistance.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_mode_support.xsd:245]:
```suggestion
					<xsd:documentation>DEPRECATED. Use mode rail with RailSubmode longDistance.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_support.xsd [hunk 6]
```diff
@@ -497,4 +523,203 @@ Rail transport, Roads and Road transport
...
+			<xsd:enumeration value="none"/>
+			<xsd:enumeration value="outside"/>
+			<xsd:enumeration value="inside"/>
+			<xsd:enumeration value="insideAndOutside"/>
+		</xsd:restriction>
+	</xsd:simpleType>
+	<!-- ======================================================================= -->
+	<xsd:simpleType name="DoorHandleEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for the type of handle for a door</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_support.xsd:596]:
```suggestion
			<xsd:documentation>Allowed values for the type of handle for a door.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd [hunk 7]
```diff
@@ -133,6 +133,93 @@ Rail transport, Roads and Road transport
...
+					<xsd:annotation>
+						<xsd:documentation>Identifier of a BEACON EQUIPMENT.</xsd:documentation>
+					</xsd:annotation>
+				</xsd:attribute>
+			</xsd:restriction>
+		</xsd:simpleContent>
+	</xsd:complexType>
+	<xsd:simpleType name="BeaconTypeEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for Beacon Type </xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd:163]:
```suggestion
			<xsd:documentation>Allowed values for Beacon Type.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd [hunk 8]
```diff
@@ -133,6 +133,93 @@ Rail transport, Roads and Road transport
...
+			<xsd:enumeration value="bidirectional">
+				<xsd:annotation>
+					<xsd:documentation>The beacon is able to do bidirectional transmissions.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+		</xsd:restriction>
+	</xsd:simpleType>
+	<xsd:simpleType name="BeaconProtocolEnumeration">
+		<xsd:annotation>
+			<xsd:documentation>Allowed values for beacon protocols </xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_support.xsd:202]:
```suggestion
			<xsd:documentation>Allowed values for beacon protocols.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_version.xsd [hunk 9]
```diff
@@ -424,6 +439,80 @@ Rail transport, Roads and Road transport
...
+		<xsd:sequence>
+			<xsd:element ref="AccessibilityAssessment" minOccurs="0"/>
+			<xsd:element name="BeaconType" type="BeaconTypeEnumeration" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>The type of the beacon</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="BeaconDirection" type="BeaconDirectionEnumeration" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>If the Beacon is unidirectional or bidirectional</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_version.xsd:500]:
```suggestion
					<xsd:documentation>If the Beacon is unidirectional or bidirectional.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_version.xsd [hunk 10]
```diff
@@ -424,6 +439,80 @@ Rail transport, Roads and Road transport
...
+	</xsd:complexType>
+	<xsd:group name="PassengerBeaconEquipmentGroup">
+		<xsd:annotation>
+			<xsd:documentation>Elements for a BEACON EQUIPMENT type. Beacons are under development so the Enumerations may change fast.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element ref="AccessibilityAssessment" minOccurs="0"/>
+			<xsd:element name="BeaconType" type="BeaconTypeEnumeration" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>The type of the beacon</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_version.xsd:495]:
```suggestion
					<xsd:documentation>The type of the beacon.</xsd:documentation>
```

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_support.xsd [hunk 11]
```diff
@@ -378,12 +378,17 @@ Rail transport, Roads and Road transport
 	</xsd:simpleType>
 	<xsd:simpleType name="QuayTypeEnumeration">
 		<xsd:annotation>
-			<xsd:documentation>Allowed values for QUAY Component Types.</xsd:documentation>
+			<xsd:documentation>Allowed values for QUAY Component Types. Special is the structure for rail, which allows a hierarchy. 
+			railIslandPlatform is the full platform positioned between two tracks, railPlatform is a side platform facing the track and railPlatformSector is a sector of such a railPlatform. A similar hierarchy is defined for a metro station layout.</xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_stopPlace_support.xsd:382]:
```suggestion
			<xsd:documentation>Allowed values for QUAY Component Types. Special is the structure for rail, which allows a hierarchy. railIslandPlatform is the full platform positioned between two tracks, railPlatform is a side platform facing the track and railPlatformSector is a sector of such a railPlatform. A similar hierarchy is defined for a metro station layout.</xsd:documentation>
```

File: xsd/NeTEx_publication_timetable.xsd [hunk 12]
```diff
@@ -1625,7 +1646,7 @@ Provides a general purose wrapper for NeTEx data content.</xsd:documentation>
...
+			<xsd:selector xpath=".//netex:TicketValidatorEquipment | .//netex:TicketingEquipment   | .//netex:SanitaryEquipment  | .//netex:PassengerSafetyEquipment | .//netex:RubbishDisposalEquipment    |.//netex:PassengerInformationEquipment   | .//netex:EntranceEquipment  | .//netex:LiftEquipment | .//netex:LiftCallEquipment | .//netex:EscalatorEquipment | .//netex:TravelatorEquipment | .//netex:StaircaseEquipment | .//netex:RampEquipment  | .//netex:RoughSurface  | .//netex:PlaceLighting|  .//netex:PlaceSign | .//netex:HeadingSign  | .//netex:GeneralSign   |.//netex:WaitingRoomEquipment   | .//netex:ShelterEquipment  | .//netex:SeatingEquipment | .//netex:TrolleyStandEquipment | .//netex:LuggageLockerEquipment |.//netex:CycleParkingEquipment    |.//netex:AccessVehicleEquipment    |.//netex:WheelchairVehicleEquipment"/>
```
Comment (skinkie) [inline xsd/NeTEx_publication_timetable.xsd:1649]:
```suggestion
			<xsd:selector xpath=".//netex:TicketValidatorEquipment | .//netex:TicketingEquipment   | .//netex:SanitaryEquipment  | .//netex:PassengerSafetyEquipment | .//netex:RubbishDisposalEquipment |.//netex:PassengerInformationEquipment | .//netex:EntranceEquipment  | .//netex:LiftEquipment | .//netex:LiftCallEquipment | .//netex:EscalatorEquipment | .//netex:TravelatorEquipment | .//netex:StaircaseEquipment | .//netex:RampEquipment | .//netex:RoughSurface | .//netex:PlaceLighting | .//netex:PlaceSign | .//netex:HeadingSign  | .//netex:GeneralSign |.//netex:WaitingRoomEquipment | .//netex:ShelterEquipment | .//netex:SeatingEquipment | .//netex:TrolleyStandEquipment | .//netex:LuggageLockerEquipment |.//netex:CycleParkingEquipment    |.//netex:AccessVehicleEquipment    |.//netex:WheelchairVehicleEquipment"/>
```

File: xsd/NeTEx_publication_timetable.xsd [hunk 13]
```diff
@@ -1625,7 +1646,7 @@ Provides a general purose wrapper for NeTEx data content.</xsd:documentation>
...
+			<xsd:selector xpath=".//netex:TicketValidatorEquipment | .//netex:TicketingEquipment   | .//netex:SanitaryEquipment  | .//netex:PassengerSafetyEquipment | .//netex:RubbishDisposalEquipment |.//netex:PassengerInformationEquipment | .//netex:EntranceEquipment  | .//netex:LiftEquipment | .//netex:LiftCallEquipment | .//netex:EscalatorEquipment | .//netex:TravelatorEquipment | .//netex:StaircaseEquipment | .//netex:RampEquipment | .//netex:RoughSurface | .//netex:PlaceLighting | .//netex:PlaceSign | .//netex:HeadingSign  | .//netex:GeneralSign |.//netex:WaitingRoomEquipment | .//netex:ShelterEquipment | .//netex:SeatingEquipment | .//netex:TrolleyStandEquipment | .//netex:LuggageLockerEquipment |.//netex:CycleParkingEquipment    |.//netex:AccessVehicleEquipment    |.//netex:WheelchairVehicleEquipment"/>
```
Comment (skinkie) [inline xsd/NeTEx_publication_timetable.xsd:1649]:
```suggestion
			<xsd:selector xpath=".//netex:TicketValidatorEquipment | .//netex:TicketingEquipment | .//netex:SanitaryEquipment | .//netex:PassengerSafetyEquipment | .//netex:RubbishDisposalEquipment |.//netex:PassengerInformationEquipment | .//netex:EntranceEquipment | .//netex:LiftEquipment | .//netex:LiftCallEquipment | .//netex:EscalatorEquipment | .//netex:TravelatorEquipment | .//netex:StaircaseEquipment | .//netex:RampEquipment | .//netex:RoughSurface | .//netex:PlaceLighting | .//netex:PlaceSign | .//netex:HeadingSign | .//netex:GeneralSign |.//netex:WaitingRoomEquipment | .//netex:ShelterEquipment | .//netex:SeatingEquipment | .//netex:TrolleyStandEquipment | .//netex:LuggageLockerEquipment |.//netex:CycleParkingEquipment |.//netex:AccessVehicleEquipment | .//netex:WheelchairVehicleEquipment"/>
```

File: xsd/netex_framework/netex_reusableComponents/netex_equipmentVehiclePassenger_version.xsd [hunk 14]
```diff
@@ -196,19 +196,14 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>VEHICLE Hoist can be used at VEHICLE has a hoist or lift  for wheelchairs.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="HoistOperatingRadius" type="LengthType" minOccurs="0">
```
[the remark this answers is not in the document]
Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_equipmentVehiclePassenger_version.xsd:199]: to nowhere.. It is also not used.. and not in the spec document..  @trurlurl @Joostb61 @duexw   . Problem that we need to include in 2.0?

File: xsd/NeTEx_publication_timetable.xsd [hunk 15]
```diff
@@ -1625,7 +1646,7 @@ Provides a general purose wrapper for NeTEx data content.</xsd:documentation>
...
+			<xsd:selector xpath=".//netex:TicketValidatorEquipment | .//netex:TicketingEquipment | .//netex:SanitaryEquipment | .//netex:PassengerSafetyEquipment | .//netex:RubbishDisposalEquipment |.//netex:PassengerInformationEquipment | .//netex:EntranceEquipment | .//netex:LiftEquipment | .//netex:LiftCallEquipment | .//netex:EscalatorEquipment | .//netex:TravelatorEquipment | .//netex:StaircaseEquipment | .//netex:RampEquipment | .//netex:RoughSurface | .//netex:PlaceLighting | .//netex:PlaceSign | .//netex:HeadingSign | .//netex:GeneralSign |.//netex:WaitingRoomEquipment | .//netex:ShelterEquipment | .//netex:SeatingEquipment | .//netex:TrolleyStandEquipment | .//netex:LuggageLockerEquipment |.//netex:CycleParkingEquipment |.//netex:AccessVehicleEquipment | .//netex:WheelchairVehicleEquipment"/>
```
Comment (skinkie) [inline xsd/NeTEx_publication_timetable.xsd:1649]:
```suggestion
			<xsd:selector xpath=".//netex:TicketValidatorEquipment | .//netex:TicketingEquipment | .//netex:SanitaryEquipment | .//netex:PassengerSafetyEquipment | .//netex:RubbishDisposalEquipment |.//netex:PassengerInformationEquipment | .//netex:EntranceEquipment | .//netex:LiftEquipment | .//netex:LiftCallEquipment | .//netex:EscalatorEquipment | .//netex:TravelatorEquipment | .//netex:StaircaseEquipment | .//netex:RampEquipment | .//netex:RoughSurface | .//netex:PlaceLighting | .//netex:PlaceSign | .//netex:HeadingSign | .//netex:GeneralSign |.//netex:WaitingRoomEquipment | .//netex:ShelterEquipment | .//netex:SeatingEquipment | .//netex:TrolleyStandEquipment | .//netex:LuggageLockerEquipment | .//netex:CycleParkingEquipment | .//netex:AccessVehicleEquipment | .//netex:WheelchairVehicleEquipment"/>
```

**Review by Joostb61 — APPROVED** [review]

**Review by trurlurl — APPROVED** [review]

**Review by duexw — APPROVED** [review]

**Comment by Aurige** [comment]
great :-) (I was jumping in to approve, but others were faster) 
Thanks

**Comment by juliustens-db** [comment]
Is the [changelog in the readme](https://github.com/NeTEx-CEN/NeTEx?tab=readme-ov-file#releases) not updated on purpose or is EPIAP/NeTEx 1.3 "officially" released now? 😄

**Comment by skinkie** [comment]
Accepting pull requests ;-)

**Comment by juliustens-db** [comment]
See #732

**Pull request merged**